hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/util/TestShell.java

```diff
@@ -528,4 +528,9 @@ public Boolean get() {
     shexc1.getProcess().waitFor();
     shexc2.getProcess().waitFor();
   }
+
+  @Test
+  public void testIsJavaVersionAtLeast() {
+    assertTrue(Shell.isJavaVersionAtLeast(8));
+  }
```

Comment:
This test succeeds on Java 7 since the `Shell.isJavaVersionAtLeast(8)` returns true even on Java 7. We should have a comment for justification at least.